bin/englishTest.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+
+const wrapperPath = path.join(__dirname, '../locales', 'wrapper', 'en.json');
+const corePath = path.join(__dirname, '../locales', 'core', 'en.json');
+
+// Removes all keys that have NOT_TRANSLATED values
+function isTranslated(value) {
+  if (value === '__NOT_TRANSLATED__') {
+    return false;
+  }
+  return true;
+}
+
+if (!Object.values(JSON.parse(fs.readFileSync(wrapperPath, 'utf8'))).every(isTranslated)) {
```

Comment:
can you extract both `if` conditions into a function for better readability?

Reply:
ok, I'll introduce some intermediate variables to make it more readable